airbyte-integrations/connectors/source-amplitude/source_amplitude/run.py

```diff
@@ -4,11 +4,49 @@
 
 
 import sys
+import traceback
+from datetime import datetime
+from typing import List
 
-from airbyte_cdk.entrypoint import launch
+from airbyte_cdk.entrypoint import AirbyteEntrypoint, launch
+from airbyte_cdk.models import AirbyteErrorTraceMessage, AirbyteMessage, AirbyteMessageSerializer, AirbyteTraceMessage, TraceType, Type
+from orjson import orjson
 from source_amplitude import SourceAmplitude
 
 
+def _get_source(args: List[str]):
```

Comment:
I think we need to mention in PR description clearly that this is will be a breaking change. And add info about this to cdk-migration file.

Reply:
yep that is the plan, I wrote up a migration guide that will be included in the next commit I push explaining what needs to change  in `run.py` and `source.py`.